templates/arista_eos_show_vrf.textfsm

```diff
@@ -1,20 +1,98 @@
 Value VRF (\S+)
 Value RD (\d\S+|<.+>)
+# Value List PROTOCOLS (\S+)
```

Comment:
Any reason these are commented out?

Reply:
I was originally going to include them but since my customer doesn't need them I am not doing that now, I will remove them